index.js

```diff
@@ -8,9 +8,92 @@
 
 process.stdin.resume();
 
+const events = require('lib/events');
+const tp     = require('core/transport');
+const peer   = require('core/file-peer');
+const chain  = require('core/chaindata');
+const pool   = require('core/pool');
+
+const waiter = require('network/waiter');
+
+const wait = require('util').promisify(setTimeout);
+
 (async function initServices() {
 
-    // Run every service
-    // After that run REPL
+    await wait(3500);
+
+    console.log('Node info:');
+    console.log(' - chain filename is %s', chain.FILENAME);
+    console.log(' - pool  filename is %s', pool.FILENAME);
+
+    // More than one node in network
+    if (tp.knownNodes.size > 1) {
+
+        await Promise.all([
+            syncChain(),
+            syncPool()
+        ]);
+
+        console.log('Node is synced and ready to receive new blocks');
+
+    }
+
+})().then(function runClient() {
+
+    console.log('Starting observer');
+
+    require('network/observer');
+
+    tp.on(events.NEW_BLOCK, function newBlock({block}, msg) {
+        console.log('Received block: %s', JSON.stringify(block));
```

Comment:
The output will be long.

Reply:
Nope. There's no actual block in UDP transport. 
There's only shortened version: hash, Phash and number.